01 Non blocking/src/js/apiIceAndFire.js

```diff
@@ -19,7 +19,7 @@ const getRequest = (config) => {
 };
 
 const sendGetRequest = (req, url) => {
-    req.open('get', url, true);
+    req.open('get', url, false);
```

Comment:
The idea is that this works with true.

Reply:
The idea is realized by the function added in main.js. I think it executes the two calls concurrently.

Reply:
Excuse me, i thinks with the function and the nested calls its execute secuencially.

Reply:
Yes you are right but then the exercise will be very easy ;)